### ISSUE TITLE
jsdoc 3.6.8 breaks compatibility with Node 12

### ISSUE THREAD

**Issue by feywind** [jsdoc/jsdoc#2000]
### Input code

It's not JavaScript, but a compilation error. When using Node 12 with jsdoc 3.6.7, everything works fine. When stepping up to 3.6.8 (patch), a new version of `klaw@4.0.1` is pulled in, which requires engines >= 14. `jsdoc` itself seems to require engines >= 8.x, so I'm assuming this isn't on purpose.

### Expected behavior

Two possibilities:

1) Either rev jsdoc's major so that CI tools don't pull in an incompatible breakage, or
2) Set the dependency on `klaw` back to `^3.0.0` until the next major - it looks like the last 3.x didn't change much besides `engines` when moving to 4.0.1.

### Current behavior

```
npm WARN notsup Unsupported engine for klaw@4.0.1: wanted: {"node":">=14.14.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: klaw@4.0.1
```

### Your environment

| Software         | Version |
| ---------------- | ------- |
| JSDoc            | 3.6.8
| Node.js          | v12.22.12
| npm              | 6.14.16
| Operating system | Linux

Also happens on my local Mac, but that's not surprising.

### Etc

I recognize that Node 12 is EOL now, but unfortunately we need to support some customers on it for a while longer. I'm not hoping for Node 12 support from you all :) just that the semvers work out.

Thanks!

**Comment by Lexus274** [comment]
Nice

**Comment by hegemonic** [comment]
Fixed by #2002.
